index.js

```diff
@@ -43,6 +43,6 @@ function readFile(file) {
 
 module.exports = {
 	isEmailClean: function(email) {
-		return (lists.indexOf(email) === -1);
+		return (lists.indexOf(email.trim()) === -1);
```

Comment:
`trim()` the user supplied email address when comparing it to the known-good list.